src/DataProtection/Cryptography.KeyDerivation/src/PBKDF2/Pbkdf2Util.cs

```diff
@@ -14,7 +14,7 @@ internal static class Pbkdf2Util
 
         private static IPbkdf2Provider GetPbkdf2Provider()
         {
-#if NETSTANDARD2_0 || NET461
+#if NETSTANDARD2_0 || NET462
```

Comment:
```suggestion
#if NETSTANDARD2_0 || NETFRAMEWORK
```

For here and everywhere down. This way we don't need to change all these places in the future when `net462` support is dropped.